internal/action/find.go

```diff
@@ -141,13 +144,21 @@ func (s *Action) findSelection(ctx context.Context, c *cli.Context, choices []st
 	}
 }
 
-func filter(l []string, needle string) []string {
+func filter(l []string, needle string, regex bool) ([]string, error) {
```

Comment:
The variables `regex` and the package `regexp` are a little bit confusing. Maybe you could name the parameter `reMatch` or something like that?

Reply:
Good point, renamed in 0a129b92fee1714d0de18f42f81ad91c2629e122.